app/client/cypress/integration/state/water-quality-overview/subTab.spec.js

```diff
@@ -0,0 +1,66 @@
+describe('Water Quality Overview Sub Tabs', () => {
+  beforeEach(() => {
+    cy.visit('/state/FL/water-quality-overview');
+  });
+
+  it('Navigating to a sub-tab selection that has no data results in “Water Type” dropdown saying “No Available Water Types” and the “Use” dropdown saying “No Available Uses”', () => {
+    // verify the water quality overview content loaded prior to other tests
+    cy.findByText('Water Quality', { timeout: 20000 }).should('exist');
+
+    // wait for the all web services to finish (surveys is usually slow here)
+    cy.findAllByTestId('hmw-loading-spinner', { timeout: 20000 }).should(
+      'not.exist',
+    );
+
+    // verify a tab without data
+    cy.findByTestId('hmw-swimming-tab-button').click();
+    cy.findByTestId('hmw-swimming-tab-panel').contains(
+      'No Available Water Types',
+    );
+    cy.findByTestId('hmw-swimming-tab-panel').contains('No Available Uses');
+
+    // verify a tab with data
+    cy.findByTestId('hmw-fishing-tab-button').click();
+    cy.findByTestId('hmw-fishing-tab-panel')
+      .contains('No Available Water Types')
+      .should('not.exist');
+    cy.findByTestId('hmw-fishing-tab-panel')
+      .contains('No Available Uses')
+      .should('not.exist');
+  });
+
+  it('Navigating to a sub-tab selection shows correct charts', () => {
+    // verify the water quality overview content loaded prior to other tests
+    cy.findByText('Water Quality', { timeout: 20000 }).should('exist');
+    cy.findByTestId('hmw-ecological-tab-button').click();
+
+    // wait for the all web services to finish (surveys is usually slow here)
+    cy.findAllByTestId('hmw-loading-spinner', { timeout: 20000 }).should(
+      'not.exist',
+    );
+
+    // Florida > Aquatic Life > Coastal Waters
+    // verify the pie chart is not there and the bar chart is
+    cy.findByTestId('hmw-ecological-tab-panel')
+      .contains('State statistical surveys provide an overall picture')
+      .should('not.exist');
+    cy.findByTestId('hmw-ecological-tab-panel')
+      .contains('Targeted monitoring provides information')
+      .should('exist');
+
+    // Florida > Aquatic Life > Rivers and Streams
+    // select a dropdown item that has the pie chart
+    // NOTE: had to use {force: true} since the input is covered by the
+    //  value text.
+    cy.get('#water-type-ecological').click({ force: true });
```

Comment:
I struggled with this one a bit... I'd prefer not to have to pass the force flag (https://docs.cypress.io/guides/references/error-messages.html#cy-failed-because-the-element-cannot-be-interacted-with). In the other tests where you passed it we were able to findByText.

I started with `cy.findByLabelText('Water Type:')` but see that's returning all the "Water Type" labels for the other sub tabs too (since they're in the DOM, just hidden). I also tried passing in the elements to scope it to w/ the second options argument (see the example in the docs: https://testing-library.com/docs/cypress-testing-library/intro#examples), but didn't get very far with that.

Not a super high priority, but it'd be good to look into this a bit more if you have the time...

Reply:
I found a way around this. I just added an id prop to the react-selects, which is applied to the container instead of the input. I left the inputId there for the label focus, but changed the input id to have label in the value.

Reply:
That makes sense. It's more tricky dealing with third-party components (like `react-select`) when you can't control how they render to the DOM. But you solution is much better/simpler – thanks for looking into it!